tests/test_vagrant.py

```diff
@@ -108,6 +108,7 @@ def teardown():
     """
     sys.stderr.write("module teardown()\n")
     if TD is not None:
+        subprocess.check_call("vagrant destroy -f", cwd=TD, shell=True)
```

Comment:
Can vagrant destroy -f return an error if the box is already destroyed ?

Reply:
It's always `exit 0`

```sh
[...]
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Forcing shutdown of VM...
==> default: Destroying VM and associated drives...
0
==> default: VM not created. Moving on...
0
```

Reply:
ok. so fine, for me